decidim-design/app/helpers/decidim/design/author_helper.rb

```diff
@@ -63,28 +60,42 @@ def author_sections
           {
             id: "source_code",
             contents: [
-              type: :table,
-              options: { headings: %w(Card Code Usage) },
-              items: author_table(
-                { name: "Default", url: "https://github.com/decidim/decidim/tree/develop/decidim-core/app/cells/decidim/author",
-                  usage: "https://github.com/decidim/decidim/blob/develop/decidim-core/app/cells/decidim/card_l/author.erb" },
-                { name: "Compact", url: "https://github.com/decidim/decidim/tree/develop/decidim-core/app/cells/decidim/author",
-                  usage: "https://github.com/decidim/decidim/blob/develop/decidim-blogs/app/views/decidim/blogs/posts/show.html.erb" },
-                { name: "Avatar", url: "https://github.com/decidim/decidim/tree/develop/decidim-core/app/cells/decidim/author", usage: "https://github.com/decidim/decidim/blob/develop/decidim-core/app/cells/decidim/endorsers_list/show.erb" }
-              )
+              {
+                type: :text,
+                values: [""],
+                cell_snippet: {
+                  cell: "decidim/author",
+                  args: [user_item],
+                  call_string: [
+                    'cell("decidim/author", _USER_PRESENTER_)',
+                    'cell("decidim/author", _USER_PRESENTER_, from: _AUTHORABLE_OR_COAUTHORABLE_RESOURCE_, context_actions: [:date], layout: :compact)',
+                    'cell("decidim/author", _USER_PRESENTER_, layout: :avatar)'
+                  ]
+                }
+              }
             ]
           }
         ]
       end
 
-      def author_table(*table_rows, **_opts)
-        table_rows.map do |table_cell|
-          row = []
-          row << table_cell[:name]
-          row << link_to(table_cell[:url].split("/").last, table_cell[:url], target: "_blank", class: "text-secondary underline", rel: "noopener")
-          row << link_to(table_cell[:usage].split("/").last, table_cell[:usage], target: "_blank", class: "text-secondary underline", rel: "noopener")
-          row
-        end
+      def user_item
+        Decidim::User.new(
+          organization: current_organization,
+          id: 2000,
+          email: "alan_smith@example.org",
+          confirmed_at: Time.current,
+          name: "Alan Smith",
+          nickname: "alan_smith",
+          personal_url: "https://alan.example.org",
+          about: "Hi, I am Alan",
+          accepted_tos_version: Time.current
+        ).presenter
+      end
+
+      def authored_item
+        OpenStruct.new(
+          published_at: 1.day.ago
+        )
```

Comment:
OpenStruct usage is not recommended anymore, we should avoid introducing new ones. From the [stdlib doc page](https://ruby-doc.org/stdlib-3.1.0/libdoc/ostruct/rdoc/OpenStruct.html#class-OpenStruct-label-Caveats):

> For all these reasons, consider not using [OpenStruct](https://ruby-doc.org/stdlib-3.1.0/libdoc/ostruct/rdoc/OpenStruct.html) at all.

There's even a [cop for this](https://msp-greg.github.io/rubocop/RuboCop/Cop/Style/OpenStructUse.html), that we didn't have time yet to actually enforce, but the idea is to eventually do so. 

```suggestion
      class AuthoredItem
        attr_accessor :published_at
      end
      
      def authored_item
        AuthoredItem.new(
          published_at: 1.day.ago
        )
```